src/transforms/lua/mod.rs

```diff
@@ -39,7 +39,7 @@ pub struct LuaConfigV1 {
     config: v1::LuaConfig,
 }
 
-/// Marker type for the version two of the configuration for the `lua` transform.
+/// Marker type for version two of the configuration for the `lua` transform.
```

Comment:
The `enum V1` has the same wording, should it be replaced as well?

Reply:
Yep, updated it.